app/gui/src/project-view/stores/suggestionDatabase/documentation.ts

```diff
@@ -80,15 +81,31 @@ export function documentationData(
   const groupIndex = groupName && project ? getGroupIndex(groupName, project, groups) : undefined
   const iconName = tagValue(parsed, 'Icon')
 
+  const macroFilter = isTagNamed('Macro')
+  const macros = parsed
+      .filter(macroFilter)
+      .reduce((acc, section) => {
+        const body = section.Tag.body
+        const match = body.match(/^(\S+) (.+)$/)
+        if (match) {
+          const name = match[1]
+          const description = match[2]
+          if (name && description) {
+            acc[name] = description
+          }
+        }
+        return acc
+      }, {} as {[k:string]: string})
+
+  const aliases = tagValue(parsed, 'Alias')?.trim().split(/\s*,\s*/g) ?? []
+
   return {
     documentation: parsed,
     docSummaryHtml: getDocumentationSummary(parsed),
     iconName: iconName != null ? (iconName as Icon) : undefined,
     groupIndex,
-    aliases:
-      tagValue(parsed, 'Alias')
-        ?.trim()
-        .split(/\s*,\s*/g) ?? [],
+    aliases: [...aliases, ...Object.keys(macros)].sort(),
```

Comment:
If the field is renamed to `aliasesAndMacros`, then I'd say this may stay as an actual solution (for a while)